qutebrowser/browser/commands.py

```diff
@@ -526,6 +526,35 @@ def tab_detach(self):
         cur_widget = self._current_widget()
         self._tabbed_browser.close_tab(cur_widget, add_undo=False)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window')
+    def tab_attach(self, win_id=None):
```

Comment:
If you do `win_id: int = None`, qutebrowser will take care of validating it and converting it to an integer - see https://github.com/qutebrowser/qutebrowser/blob/master/CONTRIBUTING.asciidoc#commands